src/main/java/uk/gov/hmcts/reform/hmc/model/HearingResponse.java

```diff
@@ -30,4 +30,7 @@
     @NotNull
     private Integer versionNumber;
 
+    @JsonProperty("partiesNotifiedDateTime")
+    private LocalDateTime partiesNotifiedDateTime;
```

Comment:
```suggestion
    @JsonProperty("partiesNotified")
    private LocalDateTime partiesNotifiedDateTime;
```
The JSON models have a slightly different name!